runtime/lua/vim/lsp.lua

```diff
@@ -121,28 +121,6 @@ local function validate_encoding(encoding)
       or error(string.format("Invalid offset encoding %q. Must be one of: 'utf-8', 'utf-16', 'utf-32'", encoding))
 end
 
-local function validate_command(input)
```

Comment:
To make it accessible as `vim.lsp._...`:

    function lsp._cmd_parts(input)

The `lsp.` prefix makes it available. The underscore avoids "publishing" it in docs (and tells users this is not a public API).

Doesn't need to move to `shared.lua`.